packages/website/ts/blockchain.ts

```diff
@@ -121,6 +123,9 @@ export class Blockchain {
     constructor(dispatcher: Dispatcher, isSalePage: boolean = false) {
         this._dispatcher = dispatcher;
         this._userAddress = '';
+        this._defaultGasPrice = new BigNumber(30000000000);
```

Comment:
Can we use ZeroEx.toBaseUnitAmount? It's unclear if that's 3 Gwei or 0.3 Gwei or 30 GWei

Reply:
`toBaseUnitAmount` goes from unit amount to baseUnit, i.e Ether to Wei. If we go from gwei to wei, it's also weird because the assumption is we are passing in the unit amount, not some other intermediary unit. I'll use a named constant to make this more readable.

Something like:

```
3 * GWEI_IN_WEI
```